json_examples/Architecture/Tensile_Cover.json

```diff
@@ -807,214 +831,208 @@
       4
     ],
     [
-      "Transform Select",
+      "Subdivide",
       0,
-      "List Item",
+      "Pulga Physics.001",
       0
     ],
     [
-      "List Input.002",
-      0,
-      "List Item",
-      1
+      "Subdivide",
+      1,
+      "Pulga Physics.001",
+      2
     ],
     [
-      "List Item",
+      "Math MK2",
       0,
-      "3pt Arc",
-      1
+      "Pulga Physics.001",
+      3
     ],
     [
-      "3pt Arc",
+      "List Input",
       0,
-      "Vector Math.001",
-      0
+      "Pulga Physics.001",
+      5
     ],
     [
-      "A Number",
+      "Transform Select",
       0,
-      "Vector Math.001",
-      1
+      "Pulga Physics.001",
+      6
     ],
     [
-      "3pt Arc",
-      0,
-      "UV Connection",
+      "Pulga Physics.001",
+      3,
+      "List Mask (out).002",
       0
     ],
     [
-      "Vector Math.001",
-      0,
-      "UV Connection",
-      1
+      "Reroute",
+      "Output",
+      "List Mask (out).002",
+      "mask"
```

Comment:
you should use indices, not socket name. for all sockets that aren't a Reroute node. Looks like many manual edits to .json files generate a bit of potential problem.

I might have to modify the way iojson stores these links..

Reply:
I did not edit the json manually...

Reply:
i see, then the function is treating the link differently if either of the nodes is a reroute node.. weird. ..even more pressure to modify that part of the format :)

Reply:
ooops. my bad @vicdoval  

https://github.com/nortikin/sverchok/blob/4260854806b381201e7cd2b89a1a83b8c71f94f9/utils/sv_IO_panel_tools.py#L102

that's working as expected then.

Reply:
i'll have a think about that.

Reply:
forget I said anything :)